src/main/java/nextstep/fp/StreamStudy.java

```diff
@@ -15,10 +15,7 @@ public static long countWords() throws IOException {
                 .get("src/main/resources/fp/war-and-peace.txt")), StandardCharsets.UTF_8);
         List<String> words = Arrays.asList(contents.split("[\\P{L}]+"));
 
-        long count = 0;
-        for (String w : words) {
-            if (w.length() > 12) count++;
-        }
+        long count = words.stream().filter(w -> w.length() > 12).count();
```

Comment:
메서드 체이닝 사용시 가독성을 위해 한줄에 하나의 dot(.)만 사용하시면 좋을 것 같아요 :)
```suggestion
        long count = words.stream()
                          .filter(w -> w.length() > 12)
                          .count();
```